### PR TITLE
Invitations Screens - Generate Invitation Confirmation

### PR DESCRIPTION
**Pull request by zainm97**
## Description
This PR adjust some text to alert the user that invites are being generated in the background

It also update the phlebotomy site with the new round of invites sent out

https://nhsd-jira.digital.nhs.uk/browse/GAL-599

## Type of changes

- [x] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [x] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: src/app/views/clinic_information/ClinicInformation.js [hunk 1]
```diff
@@ -60,98 +65,84 @@ class ClinicInformation extends Component {
...
   }
 
   onPageSizeChange(e) {
     this.context.setState({
       pageSize: e.target.value,
-      currentPage: 1
-    })
-  }
-
-  calculateDaysSince(date) {
```
Comment (zainm97) [inline src/app/views/clinic_information/ClinicInformation.js:127]: have been moved to helpermethod file

File: src/app/views/invitation_planning/helper.js [hunk 2]
```diff
@@ -5,18 +5,18 @@ export function sumQuintiles(quintileValues) {
...
-      return `${Number(quintile) + 1}`
+      return `${Number(quintile) + 1}`;
   }
 }
 
-export function getEndpoint(endpoint){
-  const API_ID = 'eqsnf31ud8';
-  return `https://${API_ID}.execute-api.eu-west-2.amazonaws.com/dev"/${endpoint}`
+export function getEndpoint(endpoint) {
+  const API_ID = "eqsnf31ud8";
```
Comment (Abdul-NHS) [inline src/app/views/invitation_planning/helper.js:20]: Is this intentionally left hard coded?

File: src/app/views/clinic_summary/ClinicSummary.js [hunk 3]
```diff
@@ -44,15 +45,23 @@ export default class ClinicSummary extends Component {
...
 
   async onIcbChangeHandler(e) {
     await this.context.setState({
-      icbSelected: e.target.value.replace('Participating ICB ', ''),
-      participatingICBSelected: e.target.value
+      icbSelected: e.target.value.replace("Participating ICB ", ""),
+      participatingICBSelected: e.target.value,
     });
     this.getClinicsFromIcbCode();
     this.setState({ loading: false });
```
Comment (Abdul-NHS) [inline src/app/views/clinic_summary/ClinicSummary.js:67]: will have to rebase main to include the context.setState line from 465 which defaults the list back to 10

File: src/app/views/invitation_summary/InvitationSummary.js [hunk 4]
```diff
@@ -1,65 +1,121 @@
...
+      clinicId: response.data.ClinicId.S,
+      clinicName: response.data.ClinicName.S,
+      address1: addressParts[0].trim(),
+      address2: firstWordAfterComma,
+      postcode: response.data.PostCode.S,
+      weeklyCapacity: weeklyCapacityList,
+      recentInvitationHistory: clinicInvitationHistory,
+      displayViewAllPrevInvitations: displayViewAllPrevInvitations,
+    });
+
```
Comment (Abdul-NHS) [inline src/app/views/invitation_summary/InvitationSummary.js:71]: this section needs the default page size from context state on main as well

**Review by NeerjaGoyalNHS — DISMISSED** [review]
LGTM

**Review by NeerjaGoyalNHS — APPROVED** [review]

File: src/app/views/invitation_summary/InvitationSummary.js (hunk 4 above)
Comment (Amar-NHS) [reply src/app/views/invitation_summary/InvitationSummary.js:71]: already exists on further line below

**Review by chenglawson1 — APPROVED** [review]

**Pull request merged**